examples/widgets/radio_group.nim

```diff
@@ -20,73 +20,52 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-import owlkettle, owlkettle/adw
+import owlkettle, owlkettle/[adw, playground]
 
 viewable App:
-  options: seq[string] = @["Option 0", "Option 1"]
+  options: seq[string] = @["Option 0", "Option 1", "Option 2"]
   selected: int = 1
+  sensitive: bool = true
+  sizeRequest: tuple[x, y: int] = (-1, -1) 
+  tooltip: string = "" 
+  spacing: int = 3 
+  rowSpacing: int = 6 
+  orient: Orient = OrientY 
 
 method view(app: AppState): Widget =
   result = gui:
     Window:
-      defaultSize = (400, 300)
-      
+      defaultSize = (800, 600)
       HeaderBar {.addTitlebar.}:
         WindowTitle {.addTitle.}:
           title = "Radio Group Example"
           if app.selected in 0..<app.options.len:
             subtitle = app.options[app.selected]
           else:
             subtitle = "Invalid Item"
-        
-        Button {.addLeft.}:
```

Comment:
Revert removing the buttons.

Reply:
Oh? I thought them superfluous since they were basically doing the exact same thing as autoform/playground was doing - Manipulating the options for demonstrative purposes.